ci/githubClient.sc

```diff
@@ -0,0 +1,170 @@
+#!/usr/bin/env amm
+
+import ammonite.ops._
+import ammonite.ops.ImplicitWd._
+
+import $file.utils
+import $file.awsClient
+
+import scala.util.control.NonFatal
+import scalaj.http._
+import upickle._
+
+def execute(path:String, body: String): Unit = {
+  val GITHUB_API_TOKEN =
+    sys.env.getOrElse("GIT_PASSWORD", throw new IllegalArgumentException("GIT_PASSWORD enviroment variable was not set."))
+  val GITHUB_API_USER =
+    sys.env.getOrElse("GIT_USER", throw new IllegalArgumentException("GIT_USER enviroment variable was not set."))
+
+  // Execute request
+  println(body)
+  val response = Http(s"https://api.github.com/$path")
+    .auth(GITHUB_API_USER, GITHUB_API_TOKEN)
+    .timeout(connTimeoutMs = 5000, readTimeoutMs = 100000)
+    .postData(body)
+    .asString
+    .throwError
+}
+
+/**
+ * Comment with msg on pull request with pullNumber.
+ */
+def comment(pullNumber: String, msg: String, event: String = "COMMENT"): Unit = {
+  val request = Js.Obj(
+    "body"  -> Js.Str(msg),
+    "event" -> Js.Str(event)
+    )
+  val path = s"repos/mesosphere/marathon/pulls/$pullNumber/reviews"
+  execute(path, request.toString)
+}
+
+/**
+ * Reject pull request with pullNumber.
+ */
+def reject(
+  pullNumber: String,
+  buildUrl: String,
+  buildTag: String): Unit = {
+  val msg = s"I'm building your change at [$buildTag]($buildUrl)."
+
+  comment(pullNumber, msg, "REQUEST_CHANGES")
+}
+
+/**
+ * Collect test results.
+ *
+ * @return the parsed test results.
+ */
+@main
+def collectTestResults(): Js.Arr = {
+
+  try {
+    // Join all results
+    val testResults = ls! pwd / 'target / "phabricator-test-reports" |? ( _.ext == "json")
```

Comment:
"phabricator-test-reports" ?

Reply:
I did not want to take the time to read out the JUnit XML files. Ideally we have a different report that just collects the skipped tests. This reporter would then replace https://github.com/mesosphere/marathon/blob/master/project/PhabricatorTestReportListener.scala. It should be a follow up item.

Reply:
https://jira.mesosphere.com/browse/MARATHON-7780